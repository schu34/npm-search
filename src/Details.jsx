import React from "react";
import { useInput, Text, Box } from "ink";

const Details = ({ details }) => {
  return (
    <Box flexDirection={"column"}>
      {details.split("\n").map(line => (
        <Text>{line}</Text>
      ))}
    </Box>
  );
};

export default Details;
