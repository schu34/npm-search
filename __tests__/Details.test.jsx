import React from "react";
import Details from "../src/Details";
import { render } from "ink-testing-library";

jest.mock("../src/utils");

describe("<Loading />", () => {
  it("should render loading screen", () => {
    const props = {
      selection: "express",
    };
    const { lastFrame } = render(<Details {...props} />);
    expect(lastFrame()).toMatch("getting packge details");
  });
});
