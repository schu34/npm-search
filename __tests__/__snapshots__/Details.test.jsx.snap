// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render loading screen 1`] = `[Function]`;
